src/dodal/plans/load_eiger_metadata_plan.py

```diff
@@ -0,0 +1,144 @@
+import time
+
+import bluesky.plan_stubs as bps
+from ophyd_async.core import DetectorTrigger
+from ophyd_async.fastcs.eiger import EigerDetector, EigerTriggerInfo
+
+from dodal.devices.detector import DetectorParams
+from dodal.log import LOGGER
+
+
+def load_metadata(
```

Comment:
Nit: I think `configure_and_arm_detector` is maybe a better name? Also, something similar for the file itself.